.github/workflows/production-daily.yml

```diff
@@ -102,7 +102,7 @@ jobs:
         with:
           payload-file-path: "./sentry-slack.json"
           payload-templated: true 
-          webhook:  ${{ secrets.SLACK_WEBHOOK_URL_TEST_ALERTS_SANDBOX }}
+          webhook:  ${{ secrets.SLACK_WEBHOOK_FIREFOX_IOS_DEV }}
```

Comment:
I have requested a webhook from the "Incoming Webhooks" of test-infra-alerts Slack app. This webhook URL is now in the repository secret.